tools/ansible-pr/generate_template.rb

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env ruby
+# Copyright 2017 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require 'optparse'
+require_relative '../../compile/core'
+require 'octokit'
+require 'time'
+
+# Class to handle Compile include to avoid Rubocop issues.
+class CompileClass
+  include Compile::Core
+end
+
+@options = {
+  directory: "#{Dir.pwd}/build/ansible/",
+  dry_run: false,
+  # Get all PRs from one day ago.
+  date: (Time.now - (3600 * 24)).iso8601
+}
+
+TEMPLATE = "
```

Comment:
If it were me, I'd split this into its own file.

Reply:
I was doing that originally and I felt like it was a waste because it was so short. In theory, this was meant to be such a short script that it would feel wasteful to have dependent files for it.